examples/demo_all.pp

```diff
@@ -31,6 +31,7 @@
   include ciscopuppet::demo_command_config
   include ciscopuppet::demo_domain
   include ciscopuppet::demo_interface
+  #include ciscopuppet::demo_interface_service
```

Comment:
@glennmatthews  It will be but I am working on a puppet function as part of a current US to distinguish between nexus platforms